elements/pfe-layouts/gulpfile.js

```diff
@@ -1,56 +1,103 @@
+const {
+  task,
+  src,
+  dest,
+  watch,
+  parallel,
+  series
+} = require("gulp");
+
+const browser_support = [
+  "last 2 versions",
+  "Firefox > 40",
+  "iOS > 5"
```

Comment:
@castastrophe iOS7 is unsupported as of 23 November 2018, so our oldest should be iOS8. Probs same for webrh. Thx!

Reply:
Updated